runtime/src/main/java/org/corfudb/recovery/FastObjectLoader.java

```diff
@@ -428,6 +429,8 @@ private void updateCorfuObject(ILogData logData) {
         LogEntry logEntry;
         try {
             logEntry = deserializeLogData(runtime, logData);
+        } catch (InterruptedException ie) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either re-interrupt this method or rethrow the "InterruptedException". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2142)